config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
 
+  resources :registered_applications
```

Comment:
Favor restricting routes to exposing too many routes as it may be a source of buggy behaviour or even a security vulnerability. 

Consider always defauting to using the `only: ...` option with the `resources` method to limit the routes that are generated to only those that are needed. Remove the `only:` option only once its clear that all 7 of the RESTful routes are needed.